packages/client/hmi-client/src/components/widgets/tera-chart-settings.vue

```diff
@@ -24,17 +25,55 @@
 				@update:model-value="toggleEnsembleChartOption('showIndividualModelsWithWeight', $event)"
 			/> -->
 		</template>
-		<tera-chart-control
-			:chart-config="{
-				selectedRun: 'fixme',
-				selectedVariable: selectedOptions
-			}"
-			:multi-select="true"
-			:show-remove-button="false"
-			:variables="selectOptions"
-			@configuration-change="$emit('selection-change', $event.selectedVariable, type)"
-		/>
+		<!-- Different layout for comparison charts -->
+		<template v-if="type === ChartSettingType.VARIABLE_COMPARISON">
+			<div class="settings-item">
+				<div class="content">
+					<tera-chart-control
+						:chart-config="{
+							selectedRun: 'fixme',
+							selectedVariable: selectedOptions
+						}"
+						:multi-select="true"
+						:show-remove-button="false"
+						:variables="selectOptions"
+						@configuration-change="$emit('selection-change', $event.selectedVariable, type)"
+					/>
+				</div>
+				<div class="actions">
+					<Button
+						icon="pi pi-cog"
+						text
+						rounded
+						@click="
+							$emit('open', {
+								...currentSetting,
+								name: `${currentSetting?.selectedVariables?.join(', ')}`
```

Comment:
Instead of patching existing setting with new name here, we should update the name of the setting properly when new variables are updated in `updateComparisonChartSetting` in `useChartSettings.ts`